Greeting/src/routes/HtmlGreeter.ts

```diff
@@ -0,0 +1,14 @@
+import Greeter from "./greeter";
+
+export class HtmlGreeter extends Greeter {
+  constructor(greeting: string, private tagName: string = "h1") {
```

Comment:
You have included the constructor and super call here, which is great. But I would expect the private modifier to live above the constructor, where you declare the additional properties. 

```
export class HtmlGreeter extends Greeting {
  tagName: string;

  constructor(greeting: string, tagName: string) {
    super(greeting);
    this.tagName = tagName;
  }

  greet(name: string): string {
    const message = super.greet(name);
    return `<${this.tagName}>${message}</${this.tagName}>`;
  }
}
```